editor/plugins/script_editor_plugin.cpp

```diff
@@ -1842,11 +1841,7 @@ void ScriptEditor::_update_script_colors() {
 
 		script_list->set_item_custom_bg_color(i, Color(0, 0, 0, 0));
 
-		bool current = tab_container->get_current_tab() == c;
-		if (current && highlight_current) {
-			script_list->set_item_custom_bg_color(i, EditorSettings::get_singleton()->get("text_editor/script_list/current_script_background_color"));
```

Comment:
How does the script list look like with this removed? Highlighting the current script seems pretty important to me to know what you're editing.

Reply:
The script highlight was actually being done twice – the code being present there was a bug all along :slightly_smiling_face: 

Removing this code makes the highlight only appear once, as the script editor plugin already applies some highlight on its own.